tests/integration.go

```diff
@@ -6,11 +6,14 @@ import (
 	"strings"
 	"testing"
 
+	"github.com/Masterminds/semver/v3"
 	"github.com/containerd/continuity/fs/fstest"
 	"github.com/moby/buildkit/util/testutil/integration"
 	"github.com/stretchr/testify/require"
 )
 
+const defaultBuildKitVersion = "v0.13.0"
```

Comment:
this should be `buildx-stable-1`. 

If this works then we can add extra workflow that tests specific version, or master.

Reply:
I need semver to properly match buildkit versions and skip related tests, see last commit: https://github.com/docker/buildx/pull/2322/files#diff-6db6da28b382b90cebe937a99fedeb6c8927535c220fd1c9a40f644d4ae90ee7R82-R92

I guess I could run the image to extract buildkitd version but that sounds a bit overkill.